### PR TITLE
feat: #WB2-1541, add toasts actions create delete edit

### PR DESCRIPTION
**Pull request by Headan**
## Describe your changes

Ajout des toasts sur les différentes actions : 

- Enregistrer une page 
- Restauration d’une version
- Duplication
- Suppression
- Création / Suppression / Edition d’un commentaire 

## Issue ticket number and link

Ticket : https://edifice-community.atlassian.net/browse/WB2-1541

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/services/queries/comment.ts [hunk 1]
```diff
@@ -4,9 +4,11 @@
 
 import { useMutation, useQueryClient } from '@tanstack/react-query';
 import { pageQueryOptions, wikiService } from '..';
+import { getToastActions } from '~/store/toast';
 
 export const useCreateComment = () => {
   const queryClient = useQueryClient();
+  const { addToastMessage } = getToastActions();
```
Comment (ccreusat) [inline frontend/src/services/queries/comment.ts:11]: tu peux appeler directement la version "hook" dans le hook react query plutôt que la version vanilla js, par exemple si tu as un `useToastActions`

File: frontend/src/store/toast/index.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      toastMessages: [...state.toastMessages, { ...message }],
+    })),
+  clearToastMessages: () => set({ toastMessages: [] }),
+}));
+
+// Selectors
+const selectToastMessages = (state: ToastState) => state.toastMessages;
+
+// Getters
+export const getToastActions = () => toastStore.getState();
```
Comment (ccreusat) [inline frontend/src/store/toast/index.ts:27]: tu peux ajouter une version "hook", pour appeler dans les queries

**Review by ccreusat — COMMENTED** [review]
quelques commentaires

File: frontend/src/hooks/useRevision/useRevision.ts [hunk 3]
```diff
@@ -5,13 +5,15 @@ import { useNavigate, useParams } from 'react-router-dom';
...
 import { useUserRights } from '~/store';
+import { getToastActions } from '~/store/toast';
 
 export const useRevision = () => {
   const toast = useToast();
   const queryClient = useQueryClient();
   const navigate = useNavigate();
   const params = useParams();
   const userRights = useUserRights();
+  const { addToastMessage } = getToastActions();
```
Comment (ccreusat) [inline frontend/src/hooks/useRevision/useRevision.ts:16]: ça vaut aussi ici, tu peux remplacer par le hook

**Review by ccreusat — COMMENTED** [review]
reste un endroit

File: frontend/src/hooks/useRevision/useRevision.ts (hunk 3 above)
Comment (Headan) [reply frontend/src/hooks/useRevision/useRevision.ts:16]: Ah oui 👍

**Pull request merged**